src/utils/utils.ts

```diff
@@ -189,3 +189,20 @@ export const exampleWrapperStyles = {
   overflow: "hidden",
   width: "100%",
 };
+
+/** Return just the text from within a JSX.Element */
+export const getTextFromElement = (
```

Comment:
@7emansell and @EdwinGuzman – Rather than reworking the logic around the `aria-label`, I added a utility function to extract plain text from an `Element`. This ensures that the value passed to the aria-label is a string.

Reply:
Looks good so far. Can you add a few tests, specifically with multiple level of children since this is recursive.